source/classes.tex

```diff
@@ -6796,8 +6796,8 @@
 \begin{codeblock}
 struct C {
   friend std::strong_equality operator<=>(const C&, const C&);
-  friend bool operator==(const C& x, const C& y) = default; // OK, returns \tcode{x <=> y == 0}
-  bool operator<(const C&) = default;                       // OK, function is deleted
+  friend bool operator==(const C&, const C&) = default; // OK, returns \tcode{x <=> y == 0}
+  friend bool operator<(const C&, const C&) = default;  // OK, function is deleted
```

Comment:
The old version could also have been rescued by adding a trailing `const`; but I am being no longer apologetic for my love of ADL friends. :)

Reply:
As a general rule, we do want to show different coding styles in the core section of the C++ standard. (That's different for the library seciton). So, showing "the other syntax" for operator< seems perfectly fine (yes, it needs a "const").